### ISSUE TITLE
Setting MAX_SIZE_TEST

### ISSUE THREAD

**Issue by timmeinhardt** [wjf5203/VNext#19]
The original SeqFormer repository (and most previous methods) limit the test time resolution. However, for this version of SeqFormer and IDOL you did not set `INPUT.MAX_SIZE_TEST`. Was this intentionally? For SeqFormer, the README.md contains the tables produced by the old repository. By not limiting the test time resolution the results of this version should be better, right? It should be noted, that this issue prevents a fair comparison with previous methods. Or am I missing something?

**Comment by wjf5203** [comment]
Hi, thanks for your attention.

In original [SeqFormer repository](https://github.com/wjf5203/SeqFormer/blob/94272384dfa357dab6c4a6cc08af4e5073942fb5/inference.py#L144),  [IFC repository](https://github.com/sukjunhwang/IFC/blob/fb2ee4571dba4700eab3b52f10e147225b763e2a/projects/IFC/configs/base_ytvis.yaml#L38), and [Mask2Former repository](https://github.com/facebookresearch/Mask2Former/blob/9b0651c6c1d5b3af2e6da0589b719c514ec0d69a/configs/youtubevis_2019/swin/video_maskformer2_swin_large_IN21k_384_bs16_8ep.yaml#L20), the INPUT.MIN_SIZE_TEST is set to 360 or 480, which means the input frames are downsampled so that the short side is 360/480 pixels, the long side is also be scaled proportionally.
Following this setting, we set INPUT.MIN_SIZE_TEST to 360 in this repository by default for both SeqFormer and IDOL.

In order to achieve the best results of IDOL reported in the paper (noted by  $\dagger$), we need to infer at 480p resolution according to the description in Table 3.

For SeqFormer, the tables are produced by 360p resolution, same with the old repository. So the result (mAP) is about the same. In our experiments, increasing the inference resolution did not improve the mAP for SeqFormer.

**Comment by timmeinhardt** [comment]
I misunderstood how `MIN_SIZE_TEST` and `MAX_SIZE_TEST` work. However, if the latter is not set some images might be scaled to have their shorter side equal `MIN_SIZE_TEST` but the longer side is still larger than 680. The original SeqFormer repo also sets the maximum for these cases. This one and IDOL do not, i.e., some sequences with specifically large widths will be processed on a higher resolution.

**Comment by wjf5203** [comment]
Yes, if MAX_SIZE_TEST is not set,  the longer side may be larger than for example 680. But the original SeqFormer repo dose not limit the maximum value of the longer side during inference, it only limits the shorter side [here](https://github.com/wjf5203/SeqFormer/blob/94272384dfa357dab6c4a6cc08af4e5073942fb5/inference.py#L144), and the  [dataloader](https://github.com/wjf5203/SeqFormer/blob/94272384dfa357dab6c4a6cc08af4e5073942fb5/datasets/ytvos.py#L241) is not used for inference.

**Comment by timmeinhardt** [comment]
Thank you for the clarification. The SeqFormer setup makes sense now. However, is it common to set `MIN_SIZE_TEST=720` for OVIS? Is this the common practice for OVIS? This means the IDOL method tests on OVIS with a significantly larger input resolution. Did you train and test IFC and SeqFormer with the same larger resolutions for the comparison in Table 5 of the IDOL paper?

**Comment by wjf5203** [comment]
At the time IDOL was done, OVIS was not popular, many algorithms were not evaluated on it, and existing algorithms performed very poorly. We think this is because the benchmark is not well explored, so we tried different resolutions and found that 720p performed best on OVIS in frame oracle experiment, which means inference on 720p gets better mask quality. So we set MIN_SIZE_TEST=720 for OVIS to ensure the high-resolution frames are fully utilized. We train and test IFC and SeqFormer with the same larger resolutions to ensure a fair comparison on OVIS in Table 5.
